boot/zephyr/pm.yml

```diff
@@ -81,7 +81,12 @@ mcuboot_primary_1:
 
 #if (CONFIG_NRF53_MULTI_IMAGE_UPDATE)
 mcuboot_secondary_1:
+#if (CONFIG_PM_EXTERNAL_FLASH_MCUBOOT_SECONDARY_1)
   region: external_flash
+#endif /* CONFIG_PM_EXTERNAL_FLASH_MCUBOOT_SECONDARY_1 */
   size: CONFIG_NRF53_RAM_FLASH_SIZE
+  placement:
+    after: mcuboot_secondary
```

Comment:
Isn't this unproper for if `CONFIG_PM_EXTERNAL_FLASH_MCUBOOT_SECONDARY_1` is selected?

Reply:
Why? You have both secondary in external flash and placing the secondary_1 after secondary makes sense.

Reply:
The first sentence was obvious. The second one sentence describes something i could anticipate, but didn't know for sure.